### DISCUSSION TITLE
Deleting all rows of last table page blocks user from continuing

### DISCUSSION THREAD

**Discussion by robbedekien** [laravel/nova-issues#4438] [Ideas & Feature Requests]
- Laravel Version: 9.17
- Nova Version: 4.9
- PHP Version: 8.1.5

### Description:
First off, if this should be a discussion, my apologies and feel free to transform this to a discussion.

When Deleting all remaining rows of the last page of a table the user gets the typical `No :resource matched the given criteria.` message and cannot navigate back to a previous page as the table thinks there are no items of that resource at all.
I understand why this is happening because the `resource_page` query parameter is set. You can easily replicate this by setting the parameter to a value that is higher than the number of pages.

This also happens with destructive actions that have been called for all items on the last page.
Would it be possible to implement a fallback either in the front or backend when trying to access a page that is higher than the number of available pages?

### Detailed steps to reproduce the issue on a fresh Nova installation:

1. Go to a table such as the resource index page with multiple pages
2. Go to the last page of that table
3. Delete all items on that page
4. The page refreshes with a message that no resources have matched the given criteria

Alternative method:
1. Go to a table such as the resource index page with multiple pages
2. Manually enter the query parameter `resource_page=x` with x being a value greater than the number of available pages
4. The page loads with a message that no resources have matched the given criteria

**Comment by robbedekien** [comment]
@crynobone have you perhaps been able to take a look at this?

**Comment by SimonBroekaertEndare** [comment]
@crynobone could this get another look please? This current UX is a blocking problem for our Client. Or is there any way around this issue?
